packages/botonic-plugin-contentful/src/contentful/manage/manage-entry.ts

```diff
@@ -170,4 +179,27 @@ export class ManageContentfulEntry {
       throw new ResourceNotFoundCmsException(contentId, e)
     }
   }
+
+  private convertValueType(key: ContentFieldType, field: any): any {
+    if (key === ContentFieldType.BUTTONS_STYLE) {
+      return field ? 'QuickReplies' : 'Buttons'
```

Comment:
this have 2 issues: 
1) it relies on the implicit fact that ButtonStyle.QUICK_REPLY has value 1. It's difficult to understand and might break if we change the enum values.
2) as you can see at https://github.com/hubtype/botonic/blob/master/packages/botonic-plugin-contentful/src/contentful/contents/text.ts#L52 we support that the field is undefined (when user has not yet selected it). This is useful because some bots want to select the style dynamically when it has not been set.
so you'll need to treat the 2+1 cases individually